core/src/main/java/cz/xtf/core/bm/BinaryBuild.java

```diff
@@ -108,10 +109,10 @@ public void update(OpenShift openShift) {
 
 	@Override
 	public void delete(OpenShift openShift) {
-		openShift.imageStreams().withName(is.getMetadata().getName()).cascading(false).withGracePeriod(0).delete();
-		openShift.buildConfigs().withName(bc.getMetadata().getName()).cascading(true).withGracePeriod(0).delete();
+		openShift.imageStreams().withName(is.getMetadata().getName()).withPropagationPolicy(DeletionPropagation.BACKGROUND).delete();
+		openShift.buildConfigs().withName(bc.getMetadata().getName()).withPropagationPolicy(DeletionPropagation.BACKGROUND).delete();
```

Comment:
Is it semantically identical? I mean that image streams were deleted with `cascading(false)` and build config with `cascading(true)` however `....withPropagationPolicy(DeletionPropagation.BACKGROUND)` is the same for both.

Reply:
It's definitely not semantically identical. `Background` policy is more or less equal of `cascading` set to `true` while If we'd like to keep the code semantically identical we'd have to set deletion option of IS to `Orphan` [1]. I wanted to unify resource deletion approach in the method (if I recall correctly then the current implementation is a result of work-arounding previous versions of the client).

Anyway this code can be left out because `Background` policy is now automatically set from Fabric 4.10.0. Just a note that `cascading` method was deprecated.

[1] https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/

Reply:
Thanks for explanation. I cannot figure out where it might be an issue so marking as lgtm.